src/main/java/racingcar/controller/GameController.java

```diff
@@ -0,0 +1,59 @@
+package racingcar.controller;
+
+import racingcar.model.Car;
+import racingcar.view.ResultView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static racingcar.utils.NumberGenerator.randomNumber;
+
+public class GameController {
+
+    private List<Car> cars;
```

Comment:
```suggestion
    private final List<Car> cars;
```
컬렉션의 참조값도 final로 막아줄 수 있을 것 같네요